component/site/models/registration.php

```diff
@@ -35,7 +35,21 @@ public function __construct()
 
 	public function store(&$data)
 	{
-		$captcha = Redshop\Helper\Utility::checkCaptcha($data);
+        $plugin = JPluginHelper::getPlugin('captcha','recaptcha');
+
+        $dataCaptcha = $data;
+
+        if ($plugin)
+        {
+            $params  = new JRegistry($plugin->params);
+
+            if ($params->get('version', '') === '2.0')
```

Comment:
@NguyenBao10 what happen if version of plugin is `2.0.1`?

Reply:
its version of recapthca and at the current, it just has 2 fixed option (1.0 and 2.0). If recaptcha releases a new version in the future, we will check the plugin again and fix later